pkgs/by-name/ce/censor/package.nix

```
@@ -0,0 +1,58 @@
+{
+  lib,
+  fetchFromGitea,
+  python3Packages,
+  wrapGAppsHook4,
+  gobject-introspection,
+  libadwaita,
+  meson,
+  ninja,
+  pkg-config,
+  desktop-file-utils,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "censor";
+  version = "0.3.0";
+  pyproject = false;
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "censor";
+    repo = "Censor";
+    rev = "v${version}";
```

```suggestion
    tag = "v${version}";
```
use tag here instead: https://github.com/NixOS/nixpkgs/blob/master/doc/build-helpers/fetchers.chapter.md

---

fixed :)